db/single/tiff/image_list.txt

```diff
@@ -11,3 +11,12 @@
 0/cat-2184682_640.tiff 0
 0/cat-3113513_640.tiff 0
 various_encoding/cat-1046544_640_zstd.tiff 1
+orientation/cat-1046544_640_horizontal.tiff
```

Comment:
Don't put the names here _yet_ - this file is used by the FileReader tests - the orientation is ignored now, so it will just make existing tests slower.

Reply:
Ok, I removed the image lists